iam-role-nlb.tf

```diff
@@ -2,15 +2,15 @@ module "eks_iam_role_nlb" {
   source                        = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
   version                       = "5.5.0"
   create_role                   = true
-  role_name                     = "cluster-nlb"
+  role_name                     = local.k8s_nlb_service_account_name
```

Comment:
@hervelemeur not blocking here but I find confusing that the variable says "service account" while in this line, it is not a Kubernetes service account but an AWS IAM role (which has to be mapped to this service account).

Using the same value is useful though!

Reply:
@dduportal I had the same thought, any suggestion for a better name?

Reply:
`nlb_account_name` (discussed the 3 of us)